src/main/java/com/zufar/onlinestore/common/response/ApiResponse.java

```diff
@@ -0,0 +1,23 @@
+package com.zufar.onlinestore.common.response;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.LocalDateTime;
+
+public record ApiResponse<T>(
+
+        @JsonProperty("data")
+        T data,
+
+        @JsonProperty("message")
+        String message,
+
+        @JsonProperty("status")
+        Integer status,
+
+        @JsonProperty("time_stamp")
```

Comment:
it must be camelCase.
 `@JsonProperty("timestamp")`

Reply:
Done